docs/config_reference.rst

```diff
@@ -1637,7 +1637,7 @@ Processor Info
 
 .. versionadded:: 3.5.0
 
-A *processor info object* in ReFrame's configuration is used to hold information about the processor of a system partition and is made available to the tests through the :attr:`processor <reframe.core.systems.SystemPartition.processor>` attribute of the :attr:`current_partition <reframe.core.pipeline.RegressionTest.current_partition>`.
+A *processor info object* in ReFrame's configuration is used to hold information about the processor of a system partition and is made available to the tests through the :attr:`processor <reframe.core.systems.SystemPartition.processor>` attribute of the :attr:`current_partition <reframe.core.pipeline.RegressionTest.current_partition>`. Note that "logical CPUs" is the number of threads, so for example :attr:`num_cpus_per_core <reframe.core.systems.SystemPartition.processor.num_cpus_per_core>` can be larger than one if a hyper-threading-like system is enabled.
```

Comment:
The comment you added is okay for me but I this it fits better ~15 lines below 
```rst
.. attribute:: .num_cpus

   :required: No
   :default: ``None``

Number of logical CPUs. This is the number of threads, so for example ...
```
What do you think?

Reply:
I thought about putting it at the top of the section because logical CPUs is used multiple times, but I'm happy to move it if you think that's better.

Reply:
hm yes, maybe it makes more sense in the top of the section, you have a point